src/it/scala/temple/generate/database/database.scala

```diff
@@ -0,0 +1,216 @@
+        Column("name"),
+        Column("isStudent"),
+        Column("dateOfBirth"),
+        Column("timeOfDay"),
+        Column("expiry")
+      ),
+      Some(
+        Conjunction(
+          Comparison("Users.id", Equal, "123456"),
+          Comparison("Users.expiry", LessEqual, "2")
+        )
+      )
+    )
+
+    val readStatementWithWhereDisjunction = Read(
+      "Users",
+      List(
+        Column("id"),
+        Column("bankBalance"),
+        Column("name"),
+        Column("isStudent"),
+        Column("dateOfBirth"),
+        Column("timeOfDay"),
+        Column("expiry")
+      ),
+      Some(
+        Disjunction(
+          Comparison("Users.id", NotEqual, "123456"),
+          Comparison("Users.expiry", Greater, "2")
+        )
+      )
+    )
+
+    val readStatementWithWhereInverse = Read(
+      "Users",
+      List(
+        Column("id"),
+        Column("bankBalance"),
+        Column("name"),
+        Column("isStudent"),
+        Column("dateOfBirth"),
+        Column("timeOfDay"),
+        Column("expiry")
+      ),
+      Some(
+        Inverse(
+          Comparison("Users.id", Less, "123456")
+        )
+      )
+    )
+
+    val readStatementComplex = Read(
+      "Users",
+      List(
+        Column("id"),
+        Column("bankBalance"),
+        Column("name"),
+        Column("isStudent"),
+        Column("dateOfBirth"),
+        Column("timeOfDay"),
+        Column("expiry")
+      ),
+      Some(
+        Conjunction(
+          Disjunction(
+            IsNull(Column("isStudent")),
+            Comparison("Users.id", GreaterEqual, "1")
+          ),
+          Disjunction(
+            Inverse(IsNull(Column("isStudent"))),
+            Inverse(Comparison("Users.expiry", Less, "TIMESTAMP '2020-02-03 00:00:00+00'"))
+          )
+        )
+      )
+    )
+
+    val insertStatement = Insert(
+      "Users",
+      List(
+        Column("id"),
+        Column("bankBalance"),
+        Column("name"),
+        Column("isStudent"),
+        Column("dateOfBirth"),
+        Column("timeOfDay"),
+        Column("expiry")
+      )
+    )
+
+    val deleteStatement = Delete(
+      "Users"
+    )
+
+    val deleteStatementWithWhere = Delete(
+      "Users",
+      Some(
+        Comparison("Users.id", Equal, "123456")
+      )
+    )
+
+    val dropStatement = Drop(
+      "Users",
+      ifExists = false
+    )
+
+    val dropStatementIfExists = Drop(
+      "Users"
+    )
+
+    val insertDataA: List[PreparedVariable] = List(
+      PreparedVariable.IntVariable(3),
+      PreparedVariable.FloatVariable(100.1f),
+      PreparedVariable.StringVariable("John Smith"),
+      PreparedVariable.BoolVariable(true),
+      PreparedVariable.DateVariable(Date.valueOf("1998-03-05")),
+      PreparedVariable.TimeVariable(Time.valueOf("12:00:00")),
+      PreparedVariable.DateTimeTzVariable(Timestamp.valueOf("2020-01-01 00:00:00.0"))
+    )
+
+    val insertDataB: List[PreparedVariable] = List(
```

Comment:
Probs need to merge develop into this branch and update the UPDATE and DELETE tests.